src/index.jsx

```diff
@@ -21,15 +19,9 @@ if (!__DEV__) {
   });
 }
 
-const { store } = storeConfigs;
-
 const root = document.getElementById('root');
 
-const Main = hot(() => (
-  <Provider store={store}>
-    <App />
-  </Provider>
-));
+const Main = hot(() => <App />);
```

Comment:
So <Provider>'s only purpose is to provide the redux store?

Reply:
The store provider, yes